client_internals_it_test.go

```diff
@@ -421,6 +423,32 @@ func clientInternalEncodeDataTest(t *testing.T) {
 	})
 }
 
+func clientClusterConnectionConfigRetryTimeTest(t *testing.T) {
+	// TODO: Adapt this test for t.Parallel()
+	//t.Parallel()
+	ctx := context.Background()
+	const ASSERTION_SECONDS = 30
+	port := it.NextPort()
+	cls := it.StartNewClusterWithOptions("dev", port, 1)
+	defer cls.Shutdown()
+	config := cls.DefaultConfig()
+	config.Cluster.ConnectionStrategy.Retry.InitialBackoff = math.MaxInt32
```

Comment:
Could you change this line to `types.Duration(math.MaxInt32 * time.Millisecond)` instead of `math.MaxInt32`? `InitialBackOff` in the Go client is in nanoseconds. Same comment for the `MaxBackoff` in the line below.

Reply:
Fixed at https://github.com/hazelcast/hazelcast-go-client/pull/861/commits/830865005e9c8f61b0599f85a7fb25de30d862fc